packages/legend-graph/src/graph/MetaModelConst.ts

```diff
@@ -42,6 +42,14 @@ export enum ROOT_PACKAGE_NAME {
   PROJECT_DEPENDENCY_ROOT = 'PROJECT_DEPENDENCY_ROOT',
 }
 
+export enum ELEMENT_ORIGIN {
```

Comment:
can we maybe use `ROOT_PACKAGE_NAME` instead to not have to create yet another categorization that is semantically equivalent?